.chloggen/fix-public-api-otlpreceiver.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: otlpreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: the public API for creating the otlpreceiver configuration for HTTPServerSettings was private. This change makes it public.
+
+# One or more tracking issues or pull requests related to the change
+issues: [#8176, #8175]
```

You should just put the number here. The issue number alone is okay (the PR will be linked)
```suggestion
issues: [8175]
```